README.win32

```diff
@@ -41,7 +41,8 @@ additional software to run (other than what came with your operating
 system).  Currently, this port is capable of using one of the
 following compilers on the Intel x86 and x86_64 architectures:
 
-      Microsoft Visual C++    version 12.0 or later
+      Microsoft Visual C++    version 14.0 or later
+                              Visual C++ 2015 or later
```

Comment:
Perhaps it would be better to phrase it as "2015 (14.0) or later". The current wording kind of implies those are two different products.

Reply:
I've adjusted that and removed a 2013 mention from `README.win32`